condensation-forum/application.py

```diff
@@ -27,6 +28,8 @@
 config = ConfigLoader("config.local.json")
 # Enable encrypted session, required for OAuth to stick
 application.secret_key = config.get("sessionSecret")
+#used for form validation
+application.config["SECRET_KEY"]=config.get("sessionSecret")
```

Comment:
Why is the sessionSecret set in two different ways here? Is it needed somewhere else?